src/app/content/highlights/components/ColorPicker.tsx

```diff
@@ -64,7 +64,7 @@ const ColorButton = styled(({className, size, style, ...props}: ColorButtonProps
 // tslint:disable-next-line:variable-name
 const ColorPicker = ({className, ...props}: Props) => {
 
-  return <div className={className}>
+  return <div className={className} data-analytics-region='highlight-color-picker'>
```

Comment:
this is nice, but i don't think anything in here is going to trigger any events will it?